src/extensions/workspace/workspace.ts

```diff
@@ -571,4 +573,22 @@ export default class Workspace implements ComponentFactory {
     });
     return Promise.all(resolveMergedExtensionsP);
   }
+
+  /**
+   * This will mutate the original extensions list and make sure all extensions has the ids with the scope / default scope
+   *
+   * @param {ExtensionDataList} extensions
```

Comment:
@GiladShoham , there three lines of documenting the parameter lists, and the return type are redundant and better be removed. Often, the parameters are changed or their types are changed but the comment remains with the old data, which leads to confusion. It's better to rely on the code only and not on the comments.